Website/src/platform/CustomFields/DemoFieldTypeFactoryStore.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+using System.Collections.Generic;
+using Sitecore.Services.GraphQL.Content.TemplateGeneration.FieldMapping;
+using Sitecore.Services.GraphQL.EdgeSchema.GraphTypes.FieldTypes;
+using Sitecore.Services.GraphQL.EdgeSchema.TemplateGeneration.FieldMapping;
+
+namespace Sitecore.Demo.Edge.Website.CustomFields
```

Comment:
This class is not a custom field. Can we please change its namespace and folder.